plugwise_usb/nodes/helpers/pulses.py

```
@@ -515,8 +520,64 @@
+                        self._logs[address][slot].is_consumption = True
+                        if self._logs[prev_address][prev_slot].is_consumption:
+                            self._logs[prev_address][prev_slot].is_consumption = False
+                            self._reset_log_references()
+                    elif self._log_production is None:
+                        self._log_production = False
+                return prev_timestamp
+
+            if self._first_prev_log_processed:
+                self._first_prev_log_processed = False
+                return None
+            
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)